skimage/metrics/_structural_similarity.py

```diff
@@ -9,9 +9,11 @@
 __all__ = ['structural_similarity']
 
 
-def structural_similarity(im1, im2, win_size=None, gradient=False,
-                          data_range=None, multichannel=False,
-                          gaussian_weights=False, full=False, **kwargs):
+def structural_similarity(im1, im2,
```

Comment:
I thought we didn't like im or img in signatures and prefered image. Should we fix here?

Reply:
Correct, just waiting for the discussion above to settle...

Reply:
Noticing these `im1, im2` as I'm reviewing #6612 and... it hurts my eyes! :cry: 

I'm adding this item to the list of [Pending API changes for 1.0](https://github.com/scikit-image/scikit-image/issues/5439#issuecomment-1268429149).